react-with-json-logic/src/models/questionnaire.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  Questionnaire as QuestionnaireBase,
+  AnyQuestion as AnyQuestionBase,
+  ResultCategory as ResultCategoryBase,
+  Result as ResultBase,
+  Variable as VariableBase,
+  QuestionnaireMeta as QuestionnaireMetaBase,
+  ISOLanguage,
+} from "covquestions-js/models/Questionnaire.generated";
+
+export interface Questionnaire extends QuestionnaireBase {
```

Comment:
I wonder if we should rename our extended types instead of the base types, since both are used in parts of the editor code and it might be confusing which is use when.

Maybe EditorQuestionnaire, EditorAnyQuestion, etc